_includes/navigation-sliding.html

```diff
@@ -1,13 +1,20 @@
 <nav role="navigation" id="js-menu" class="sliding-menu-content">
   <h5>{{ site.title }} <span>{{ site.data.messages.locales[site.locale].toc }}</span></h5>
   <ul class="menu-item">
-    {% for link in site.data.navigation %}<li>
-      <a href="{{ site.url }}{{ link.url }}">
-        {% if link.image %}<img src="{{ site.url }}/images/{{ link.image }}" alt="teaser" class="teaser">{% endif %}
-        <div class="title">{{ link.title }}</div>
-        {% if link.excerpt %}<p class="excerpt">{{ link.excerpt }}</p>{% endif %}
-      </a>
-    </li>{% endfor %}
+    {% for link in site.data.navigation %}
+      <li>
+        {% if link.url contains 'http' %}
```

Comment:
Is there a `startswith` operation or similar? Just to be thorough...

Reply:
Let me check

Reply:
No, there are no such operators. Going through the Liquid docs, it doesn't seem I can make a custom one either.

Reply:
I could write a custom filter or tag though.

Reply:
https://github.com/Maritime-Robotics-Student-Society/blog/pull/18/commits/0a2988acfb269ab21d08f4d42ff4db7c498221c4
^ This should solve it. I've tested it on my machine and it works on my end.

I'll work on a generator for creating the anchors later.

Reply:
https://github.com/Maritime-Robotics-Student-Society/blog/pull/18/commits/2cd35c2072aa46f2129170583739e9d2945d2735 also handles the Github link, where we want the browser to open a new tab when we click on it. Admittedly not the most elegant solution.